activerecord/lib/active_record/associations.rb

```diff
@@ -1250,6 +1250,9 @@ module ClassMethods
         # [collection.create!(attributes = {})]
         #   Does the same as <tt>collection.create</tt>, but raises ActiveRecord::RecordInvalid
         #   if the record is invalid.
+        # [collection.reload]
+        #   Returns an array of all of the associated objects, forcing a database read.
```

Comment:
Yes, that documentation is also wrong. It was last touched by https://github.com/rails/rails/commit/6e754551 9 years ago when the behavior was different. I would rephrase it by replacing `array` with `Relation`